numpy/core/_dtype_ctypes.py

```diff
@@ -33,17 +33,43 @@ def _from_ctypes_array(t):
 
 
 def _from_ctypes_structure(t):
-    # TODO: gh-10533, gh-10532
-    fields = []
+    # TODO: gh-10533
     for item in t._fields_:
         if len(item) > 2:
             raise TypeError(
                 "ctypes bitfields have no dtype equivalent")
-        fname, ftyp = item
-        fields.append((fname, dtype_from_ctypes_type(ftyp)))
+    else:
```

Comment:
No point in this else - you can remove it and unindent everything below

Reply:
Indeed. Fixed.